### ISSUE TITLE
Marlin G7

### ISSUE THREAD

**Issue by ghost** [openhardwarecoza/LaserWeb3#53]
@HakanBastedt - for now I removed your G7 raster code from laserraster.js but I put it in safekeeping.  There has been a lot of changes to laserraster.js and raster.js over the last few weeks so we'll need to work those back into G7Rasterizer.

While looking at the rotary parsing, I only now noticed that you had some G7 parsing in the 3D viewer. https://github.com/openhardwarecoza/LaserWeb3/blob/master/public/lib/gcode-viewer/gcode-parser.js#L996  

Whats the state of that? Or let me rephrase,  once the marlin buffer work is done,  what else was left to get G7 engrave working nicely for the Marlin folks?

**Comment by MetalMusings** [comment]
Ok to remove the g7 code. It wasn't supposed to be there.

Marlin serial use pingpong and as far as i remember it worked. Not tested a lot. It might need a review. Have focused on other projects.
